update based on customer query (#6644)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/retention-policies.md

That page has additional information about creating and managing retention policies for other workloads in Microsoft 365. For example, you might want to also create a retention policy for Microsoft 365 Groups to retain and delete files that are accessed in Teams and stored in OneDrive or SharePoint.  

## Retention of edited messages

> Retention on Teams messages works on the timestamp of creation of original message. Lets take a scenario- If there is a policy of 30-day deletion on Teams chat or channel messages and a user sends a message (“Original message”) on day -1, later modifies it to “edited message”, on day 15.

Reviewer: ```suggestion
Retention on Teams messages works on the timestamp of the creation of the original message. For example, in a scenario where there is a 30-day deletion policy on Teams chat or channel messages and a user sends a message (“Original message”) on day -1, then later modifies it to an “edited message” on day 15.
```

> Here, the original message will be copied to a temporary location in the backend storage and deleted within 21 days. If there is Legal hold (via eDiscovery portal) or Retention hold in place on the user, then all versions of original, deleted and edited messages are stored as long as the hold policy is active.

Reviewer: ```suggestion
In this scenario, the original message will be copied to a temporary location in the backend storage and deleted within 21 days. If there is a Legal hold (via the eDiscovery portal) or a Retention hold in place on the user, then all versions of original, deleted, and edited messages are stored as long as the hold policy is active.
```

> The “edited message” would be deleted by the retention policy on Day-30 i.e. based on the timestamp on the original message.

Reviewer: ```suggestion
The edited message would be deleted by the retention policy on day 30, that is, based on the timestamp on the original message.
```
